src/ddsrt/src/strtod.c

```diff
@@ -174,18 +174,44 @@ ddsrt_strtod(const char *nptr, char **endptr, double *dblptr)
 dds_return_t
 ddsrt_strtof(const char *nptr, char **endptr, float *fltptr)
 {
-  /* Just use os_strtod(). */
```

Comment:
Even more convenient might be to delete it altogether!

Reply:
I wanted to clarify if you're suggesting that I just remove `ddsrt_strtof` altogether. My concern is that it would be a breaking change to remove it from the public API of `ddsrt`, but I suppose it makes some sense as Cyclone isn't actually using it for anything.

Reply:
I don't consider `ddsrt` a public interface, so yes, I was indeed suggesting removing `ddsrt_strtof` altogether.

Reply:
Okay, I removed `ddsrt_strtof` completely.